// @zeplin/sdk/dist/models/screen-section.js
"use strict";

require("core-js/modules/es.object.define-property.js");

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.transformScreenSectionToJSON = exports.transformJSONToScreenSection = void 0;

require("core-js/modules/es.function.name.js");

require("core-js/modules/es.symbol.js");

require("core-js/modules/es.symbol.description.js");

var _entityReference = require("./entity-reference");

/* tslint:disable */

/* eslint-disable */

/**
 * Zeplin API
 * Access your resources in Zeplin
 *
 * Contact: support@zeplin.io
 *
 * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
 * https://openapi-generator.tech
 * Do not edit the class manually.
 */
var transformScreenSectionToJSON = function transformScreenSectionToJSON(value) {
  return {
    id: value.id,
    created: value.created,
    name: value.name,
    description: value.description,
    parent: value.parent && (0, _entityReference.transformEntityReferenceToJSON)(value.parent)
  };
};

exports.transformScreenSectionToJSON = transformScreenSectionToJSON;

var transformJSONToScreenSection = function transformJSONToScreenSection(value) {
  return {
    id: value.id,
    created: value.created,
    name: value.name,
    description: value.description,
    parent: value.parent && (0, _entityReference.transformJSONToEntityReference)(value.parent)
  };
};
/**
 * 
 * @export
 * @interface ScreenSection
 */


exports.transformJSONToScreenSection = transformJSONToScreenSection;
